src/openrct2/drawing/ImageImporter.cpp

```diff
@@ -129,18 +132,21 @@ std::tuple<void*, size_t> ImageImporter::EncodeRLE(const int32_t* pixels, uint32
     };
 
     auto src = pixels;
-    auto buffer = static_cast<uint8_t*>(std::malloc((height * 2) + (width * height * 16)));
-    if (buffer == nullptr)
+    auto buffer = std::vector<uint8_t>();
+    buffer.resize((height * 2) + (width * height * 16));
+
+    if (buffer.data() == nullptr)
+
     {
         throw std::bad_alloc();
     }
```

Comment:
This code can be removed.  `std::vector` will already throw a  `std::bad_alloc` if it can't get the memory